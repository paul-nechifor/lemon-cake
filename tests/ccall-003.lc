(= printf-func (dynsym 'libc.so' 'printf'))
(= printf (~ (a b) (ccall printf-func a b)))

(printf 'hello%lld' 3)
(printf 'hello%lld' 8)
(printf 'hello %s end' 'str')
