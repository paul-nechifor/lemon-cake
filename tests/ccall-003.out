hello3hello8hello str end
